### PR TITLE
Release candidate 6

### PR DESCRIPTION
**Pull request by hiraldesai**
- Package upgrades
- Readme update
- Changelog update

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,11 +3,18 @@
...
+- Upgraded various package
+- Updated readme
+
+## [3.0.0-rc5] - 2019-03-18
+
+### Change
+- Upgraded the underlying Application Insights reference to the [new beta version](ai-beta)
+- Completely rewritten in TypeScript
 
 ## [Unreleased]
```
Comment (pviotti) [inline CHANGELOG.md]: you can delete this line

File: CHANGELOG.md [hunk 2]
```diff
@@ -54,4 +61,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 [2.0.0]: https://github.com/Azure/react-appinsights/compare/v1.0.4...v2.0.0
 [2.0.1]: https://github.com/Azure/react-appinsights/compare/v2.0.0...v2.0.1
 [2.0.2]: https://github.com/Azure/react-appinsights/compare/v2.0.1...v2.0.2
-[3.0.0]: https://github.com/Azure/react-appinsights/compare/v2.0.1...typescript-rewrite
+[3.0.0-rc5]: https://github.com/Azure/react-appinsights/compare/v2.0.1...TypeScript-rewrite
```
Comment (pviotti) [inline CHANGELOG.md]: this link should be `https://github.com/Azure/react-appinsights/compare/v2.0.2...v3.0.0-rc5 `

File: README.md [hunk 3]
```diff
@@ -143,3 +147,5 @@ Refer to [this doc][appinsights-js-api] for information on the Javascript API of
 [appinsights-nodejs]: https://azure.microsoft.com/en-us/documentation/articles/app-insights-nodejs/
 [appinsights-js-api]: https://github.com/Microsoft/ApplicationInsights-JS/blob/master/API-reference.md
 [react-router]: https://github.com/ReactTraining/react-router/blob/master/FAQ.md#how-do-i-access-the-history-object-outside-of-components
+[2-0-2]: https://github.com/Azure/react-appinsights/tree/v2.0.2
+[3-0-0-rc-5]: https://www.npmjs.com/package/react-appinsights/v/3.0.0-rc.5
```
Comment (pviotti) [inline README.md]: change to rc6, and also above

**Review by pviotti — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (hiraldesai) [reply README.md]: Done

**Review by pviotti — APPROVED** [review]

**Pull request merged**